types/rollup-plugin-less/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/xiaofuzi/rollup-plugin-less##readme
 // Definitions by: Tristan <https://github.com/LeoDog896>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.5
 
 /// <reference types="node" />
 /// <reference types="jsdom" />
```

Comment:
```suggestion
```
IMO this is a proper, long term fix. This looks like mistake and JSDom just `got dragged in other peoples problems`, so this is a fix for JSDom 20 going forward here:
```diff
diff --git a/types/rollup-plugin-less/index.d.ts b/types/rollup-plugin-less/index.d.ts
index 79dfb9f1f0..0190e54a48 100644
--- a/types/rollup-plugin-less/index.d.ts
+++ b/types/rollup-plugin-less/index.d.ts
@@ -4,8 +4,8 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
-/// <reference types="jsdom" />
 /// <reference types="less" />
+
 import { Plugin } from 'rollup';
 
 declare namespace less {
diff --git a/types/rollup-plugin-less/tsconfig.json b/types/rollup-plugin-less/tsconfig.json
index 30068ea395..8beb8aa6a3 100644
--- a/types/rollup-plugin-less/tsconfig.json
+++ b/types/rollup-plugin-less/tsconfig.json
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
         ],
         "noImplicitAny": true,
         "noImplicitThis": true,
```

Reply:
so it requires a change in `tsconfig` as well

Reply:
> It would be nice to get this PR merged and published, because v20 jsdom fails to this error now and is unusable

Let's merge this one, i'll fix `rollup-plugin-less` in other PR